Cargo.toml

```diff
@@ -243,7 +244,6 @@ try_err = "deny"
 lossy_float_literal = "deny"
 let_underscore_must_use = "deny"
 iter_over_hash_type = "deny"
-impl_trait_in_params = "deny"
```

Comment:
I removed this lint - IMO this is not something we need to deny

Reply:
OK but it's worth noting `impl Trait` as a param is strictly less powerful than `<T: Trait>`.